lib/ecmascript.ts

```diff
@@ -3428,9 +3415,9 @@ export function BalanceDurationRelative(
   if (sign === 0) return { years, months, weeks, days };
 
   let calendar;
-  let relativeTo: Temporal.PlainDateTime;
+  let relativeTo: Temporal.PlainDate;
```

Comment:
Could the discrepancy be that this should be typed as `Temporal.PlainDate | undefined`?

Reply:
TS type differences shouldn't cause different runtime behavior, though.

Reply:
BTW, the reason why lack of `undefined` in that declaration doesn't cause a TS error is because `strictNullChecks` is still `false`. I filed #98 to track the (considerable) work to be able to turn it on.